src/overlays/actors/ovl_Obj_Grass/z_obj_grass.c

```diff
@@ -95,42 +97,44 @@ void func_809A91FC(MtxF* matrix) {
     }
 }
 
-void func_809A92D0(ObjGrassStruct1_1* ptr, PlayState* play) {
-    if (!(ptr->unk_0E & 0x10)) {
-        Item_DropCollectibleRandom(play, NULL, &ptr->unk_00, ptr->unk_0E * 0x10);
+void ObjGrass_DropCollectible(ObjGrassElement* grassElem, PlayState* play) {
+    if ((grassElem->dropTable & 0x10) == 0) {
```

Comment:
Why the `== 0` instead of `!`?

Reply:
Probably because I made a mistake when analyzing the variable.  assumed that dI assumed that `grassElem->dropTable` stored a single value ranging from 0x00-0x1F, but it seems to consistently test `(dropTable & 0x10) == 0` before passing the value to Item_DropCollectibleRandom,